common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAuthority.java

```diff
@@ -189,6 +191,12 @@ public synchronized boolean isSameCloudAsAuthority(@NonNull final AzureActiveDir
             return false;
         }
 
+        if (cloudOfThisAuthority != null && cloudOfAuthorityToCheck != null) {
+            // Depending upon the caller of this method, home_cloud_name may or may not be a PRT's home cloud.
+            SpanExtension.current().setAttribute(AttributeName.home_cloud_name.name(), cloudOfAuthorityToCheck.getPreferredCacheHostName());
+            SpanExtension.current().setAttribute(AttributeName.requested_cloud_name.name(), cloudOfThisAuthority.getPreferredCacheHostName());
```

Comment:
How do we know this is request authority? I don't see this being passed to the method. I would think that request authority should've been an input to this method but there is just one input `authorityToCheck` which it seems you are treating as home. But how do you know the other one is request authority because it doesn't seem to come from caller or parameters?

Reply:
cloudOfAuthorityToCheck is coming from requested parameter authorityToCheck.  
cloudOfThisAuthority is coming from current object. The method is invoked on the current object which is the requestAuthority in https://github.com/AzureAD/ad-accounts-for-android/blob/367491459f7fafa542b73b4f9032ee323a05e5d2/broker4j/src/main/com/microsoft/identity/broker4j/broker/BrokerControllerFactory.java#L248 .

Reply:
Ok I see, this is an instance method. Probably worth using `this` keyword when accessing current object